### PR TITLE
feat(pci.storages.instances-backup): create instance from a backup

### PR DESCRIPTION
**Pull request by marie-j**
MANAGER-2352

### CODE REVIEW

File: packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html [hunk 1]
```diff
@@ -11,7 +11,9 @@
 
             <p data-translate="{{:: 'pci_project_flavors_category_definition_' + flavorByCategory.category }}"></p>
 
-            <div class="container-fluid px-0">
+            <p data-ng-if="flavorByCategory.flavors.length === 0" data-translate="pci_project_flavors_category_definition_none"></p>
+
+            <div class="container-fluid px-0" data-ng-if="flavorByCategory.flavors.length > 0">
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html]: Not yours but removing default padding from the container doesn't make sens `container-fluid px-0`.

File: packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html [hunk 2]
```diff
@@ -58,6 +61,12 @@
                                 </span>
                             </div>
                         </oui-select-picker-section>
+                        <oui-select-picker-section data-ng-if="$ctrl.region && !$ctrl.region.hasEnoughQuotaForFlavor(flavor)">
+                            <a class="oui-button oui-button_link"
+                               data-ui-sref="pci.projects.project.quota">
+                               <span data-translate="pci_project_flavors_quota"></span>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html]: The `translate` directive could be applied directly to the parent element.

File: packages/manager/modules/pci/src/projects/project/storages/instance-backups/add/add.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
+<section>
+    <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+    <h1>{{:: 'pci_projects_project_instances_backup_add_title' | translate }}</h1>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/instance-backups/add/add.html]: Why not prefer using the `translate` directive?

File: packages/manager/modules/pci/src/projects/project/storages/instance-backups/add/add.html [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        <oui-field
+            data-label="{{ :: 'pci_projects_project_instances_backup_add_name_label' | translate }}"
+            data-size="xl">
+            <input class="oui-input" type="text" id="name" name="name"
+                ng-model="$ctrl.instance.name"
+                ng-pattern="$ctrl.instanceNamePattern"
+                required>
+
+            <div class="oui-field__helper">
+                <span data-translate="pci_projects_project_instances_backup_add_name_help"></span>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/instance-backups/add/add.html]: Is `translate` directive could be applied to the parent element?

File: packages/manager/modules/pci/src/projects/project/storages/instance-backups/add/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/instance-backups/add/translations/Messages_fr_FR.json]: Extra empty line.

File: packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html (hunk 1 above)
Comment (jleveugle) [reply packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html]: why ?

**Pull request merged**